### PR TITLE
feat(session): make session inactivity duration configurable

### LINKED ISSUE #5395: Google Oauth JWT token is valid only for 5 minutes
While Chronograf's cookie expiration time is set correctly, the JWT expiration time is set only for five minutes, which causes Google reauthentication if the Chronograf page was closed for more than 5 minutes and then reopened.

Following lines get logged:
```
 time="2020-02-20T21:14:16Z" level=error msg="Invalid principal" component="token_auth" method=GET remote_addr="172.31.3.89:35448" url=/chronograf/v1/me 
 time="2020-02-20T21:14:16Z" level=info msg="Response: Forbidden" component=server method=GET remote_addr="172.31.3.89:35448" response_time="53.142µs" status=403 
 time="2020-02-20T21:14:16Z" level=info msg="Response: Temporary Redirect" component=server method=GET remote_addr="172.31.3.89:35448" response_time="97.954µs" status=307 
```

Payload from JWT:
```
{
  "exp": 1582281075,
  "iat": 1582280773,
  "iss": "google",
  "nbf": 1582280773,
  "sub": "oleksandr.vilchynskyy@xxxxx.com",
  "grp": "xxxxx.com"
}
```

### PR DESCRIPTION
**Pull request by localhots**
Closes #5395

Makes session inactivity duration configurable. 
Currently the default is 5 minutes and it could only go lower if session duration is below that value.

  - [ ] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -63,16 +63,17 @@ type Server struct {
...
-	ResourcesPath   string        `long:"resources-path" description:"Path to directory of pre-canned dashboards, sources, kapacitors, and organizations (/usr/share/chronograf/resources)" env:"RESOURCES_PATH" default:"canned"`
-	TokenSecret     string        `short:"t" long:"token-secret" description:"Secret to sign tokens" env:"TOKEN_SECRET"`
-	JwksURL         string        `long:"jwks-url" description:"URL that returns OpenID Key Discovery JWKS document." env:"JWKS_URL"`
-	UseIDToken      bool          `long:"use-id-token" description:"Enable id_token processing." env:"USE_ID_TOKEN"`
-	LoginHint       string        `long:"login-hint" description:"OpenID login_hint paramter to passed to authorization server during authentication" env:"LOGIN_HINT"`
-	AuthDuration    time.Duration `long:"auth-duration" default:"720h" description:"Total duration of cookie life for authentication (in hours). 0 means authentication expires on browser close." env:"AUTH_DURATION"`
+	Develop            bool          `short:"d" long:"develop" description:"Run server in develop mode."`
```
Comment (sranka) [inline server/server.go:66]: Is it possible to revert all the changes that are not related to this PR?

**Review by sranka — CHANGES_REQUESTED** [review]

File: server/server.go (hunk 1 above)
Comment (localhots) [reply server/server.go:66]: Excuse me this is just how gofmt works. I can move the new field to it's own "group" and this will restore indentation in the red block above, but new setting will appear disconnected from similarly grouped ones.

Comment (sranka) [reply server/server.go:66]: ok, understood, so it is fine then

**Review by sranka — APPROVED** [review]
looks good to me

**Comment by sranka** [comment]
@localhots thank you for your contribution, can you please also change the commit message to `feat(session): make session inactivity duration configurable`, we prefer [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/) ...

**Comment by localhots** [comment]
Done @sranka 
Anything else I need to do? I realize I didn't complete all of the checklist

**Comment by sranka** [comment]
@localhots 
Please, rebase to master (to avoid conflict) and update Features under 1.8.7 in CHANGELOG.md with:
```
1. [#5585](https://github.com/influxdata/chronograf/pull/5585): Make session inactivity duration configurable.
```

**Comment by localhots** [comment]
@sranka also done :smiley:

**Comment by sranka** [comment]
@localhots your changes are now part of 1.8.7. release. AFAIK it should happen this week. Thank you for your quick actions and contribution.

**Comment by localhots** [comment]
Thank you for such a quick response too @sranka
